blockstore/src/test/java/brooklyn/location/blockstore/AbstractVolumeManagerLiveTest.java

```diff
@@ -142,6 +153,15 @@ public void testCreateVolume() throws Exception {
     // Does the attach+mount twice to ensure that cleanup worked
     @Test(groups="Live", dependsOnMethods = "testCreateVolume")
     public void testCreateAndAttachVolume() throws Exception {
+        runCreateAndAttachVolume(ReattachMode.RE_ATTACH);
+    }
+    
+    // Does the attach+mount twice to ensure that cleanup worked
+    protected enum ReattachMode {
```

Comment:
Maybe, but given it's in test code, and only used by one test (to control whether we test detach + re-attach, which isn't possible in azure-arm currently), then I don't think it's worth trying to generalise it yet.